dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -92,6 +92,27 @@ Resources:
         log_statement: ddl
         log_statement_sample_rate: '1.0'
 
+  {{ service.prefix }}KMSKey:
+    Type: "AWS::KMS::Key"
+    Properties: 
+      Description: "KMS Key for Aurora encryption"
+      KeyPolicy:
+        Version: '2012-10-17'
+        Id: !Sub '${App}-${Env}-cluster-key'
+        Statement:
+        - Sid: Enable IAM User Permissions
+          Effect: Allow
+          Principal:
+            AWS: !Sub "arn:aws:iam::${AWS::AccountId}:root"
+          Action: kms:*
+          Resource: '*'
+
+  {{ service.prefix }}KeyAlias:
+    Type: 'AWS::KMS::Alias'
+    Properties:
+      AliasName: !Sub 'alias/${App}-${Env}-cluster-key'
```

Comment:
This will likely need `{{ service.prefix }}` to avoid dupes where end users may run multiple dbs